pkg/i18n/english.go

```diff
@@ -932,8 +942,11 @@ func EnglishTranslationSet() TranslationSet {
 		// the actual view is the extras view which I intend to give more tabs in future but for now we'll only mention the command log part
 		LcOpenExtrasMenu:                    "open command log menu",
 		LcShowingGitDiff:                    "showing output for:",
+		LcCopyCommitDiffToClipboard:         "copy commit diff to clipboard",
 		LcCopyCommitShaToClipboard:          "copy commit SHA to clipboard",
+		LcCopyCommitURLToClipboard:          "copy commit URL to clipboard",
 		LcCopyCommitMessageToClipboard:      "copy commit message to clipboard",
+		LcCopyCommitAttributeToClipboard:    "Copy commit attribute...",
```

Comment:
There's no need for the ellipsis here (...) because we're using `OpensMenu: true,` on the keybinding, so that will automatically be appended. Having a triple dot here results in six dots in a row in the menu :P